apps/ui/src/views/Townhall/Topics.vue

```diff
@@ -14,7 +15,7 @@ const {
   isError,
   isFetchingNextPage
 } = useTopicsQuery({
-  spaceId: toRef(() => props.space.id)
+  spaceId: toRef(() => props.townhallSpace.space_id)
```

Comment:
`props.townhallSpace` can be null, and result in 

```
Topics.vue:17 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'space_id')
```

Reply:
It shouldn't be null (this view expects non-nullable `townhallSpace`, but I see that we don't handle error case there, so we actually can pass `undefined`:

https://github.com/snapshot-labs/sx-monorepo/blob/cdbf72b331153136c9e3cff88da3e78961938214/apps/ui/src/views/Space.vue#L59-L67

Have you reproduced this issue? Or is there also a different case where it can be falsey?

I will update to handle that.

Reply:
Seems like the error come from Roles.vue. it's happening with http://localhost:8080/#/s:ethpoll.eth/townhall/roles

Reply:
I see, it will happen in most of those views because `ethpoll.eth` is not defined space (`openagora.eth` is now the only one that is supported. I haven't updated the test plan though.

I will add case so prop types are reflecting reality.